### PR TITLE
Janitorial Cleaning Gloves

### PR DESCRIPTION
**Pull request by FlamingLily**
image
image
image

Lore team approved :)
image

They have low permeability, but are otherwise just coloured gloves for RP purposes.

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/loadout_xeno/tajara.dm [hunk 1]
```diff
@@ -42,6 +42,14 @@
 	taj_gloves["machinist gloves"] =  /obj/item/clothing/gloves/black/tajara/smithgloves
 	gear_tweaks += new /datum/gear_tweak/path(taj_gloves)
 
+/datum/gear/gloves/tajaran_janitor
+	display_name = "tajara cleaning gloves"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_xeno/tajara.dm:46]:
```suggestion
	display_name = "tajaran cleaning gloves"
```
i think.

File: code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm [hunk 2]
```diff
@@ -69,6 +69,15 @@
...
 
+/datum/gear/gloves/unathi_janitor
+	display_name = "unathi cleaning gloves"
+	description = "A pair of rubber cleaning gloves, designed for use by Unathi."
+	path = /obj/item/clothing/gloves/janitor/unathi
+	whitelisted = list(SPECIES_UNATHI)
+	allowed_roles = list("Janitor")
+	sort_category = "Xenowear - Unathi"
+
+
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm:80]: stray newline.

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 3]
```diff
@@ -139,10 +139,32 @@
 	name = "tajaran leather gloves"
 	species_restricted = list(BODYTYPE_TAJARA)
 
-/obj/item/clothing/gloves/botanic_leather/vaurca 
+/obj/item/clothing/gloves/botanic_leather/vaurca
 	name = "vaurca leather gloves"
 	species_restricted = list(BODYTYPE_VAURCA)
 
+
```
Comment (SleepyGemmy) [inline code/modules/clothing/gloves/miscellaneous.dm:146]: stray newline.

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 4]
```diff
@@ -139,10 +139,32 @@
...
+/obj/item/clothing/gloves/janitor
+	name = "rubber cleaning gloves"
+	desc = "A pair of thick, long rubber gloves, designed to prevent the wearer from the splash of industrial strength cleaners. Not electrically insulated."
```
Comment (SleepyGemmy) [inline code/modules/clothing/gloves/miscellaneous.dm:149]:
```suggestion
	desc = "A pair of thick, long, yellow rubber gloves, designed to protect the wearer from the splash of industrial strength cleaners. Not electrically insulated."
```

Comment (SleepyGemmy) [inline code/modules/clothing/gloves/miscellaneous.dm:149]: i get that they don't protect fully against electricity for game balance reasons, but rubber is an insulator. maybe add just a little bit of protection?

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

File: code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm (hunk 2 above)
Comment (FlamingLily) [inline code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm:80]:
```suggestion
```

File: code/modules/clothing/gloves/miscellaneous.dm (hunk 3 above)
Comment (FlamingLily) [inline code/modules/clothing/gloves/miscellaneous.dm:146]:
```suggestion
```

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 5]
```diff
@@ -139,10 +139,32 @@
...
 	name = "vaurca leather gloves"
 	species_restricted = list(BODYTYPE_VAURCA)
 
+
+/obj/item/clothing/gloves/janitor
+	name = "rubber cleaning gloves"
+	desc = "A pair of thick, long, yellow rubber gloves, designed to protect the wearer from the splash of industrial strength cleaners. Not electrically insulated."
+	icon_state = "janitor"
+	item_state = "janitor"
+	permeability_coefficient = 0.01 //Prevents chemical seepage as well as latex, but without any of the sterility or protection
```
Comment (FlamingLily) [inline code/modules/clothing/gloves/miscellaneous.dm:152]:
```suggestion
	permeability_coefficient = 0.01 //Prevents chemical seepage as well as latex, but without any of the sterility or protection
	siemens_coefficient = 0.50
```
This makes them about the same as budget insuls/leather/black gloves

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 6]
```diff
@@ -139,10 +139,32 @@
...
 
-/obj/item/clothing/gloves/botanic_leather/vaurca 
+/obj/item/clothing/gloves/botanic_leather/vaurca
 	name = "vaurca leather gloves"
 	species_restricted = list(BODYTYPE_VAURCA)
 
+
+/obj/item/clothing/gloves/janitor
+	name = "rubber cleaning gloves"
+	desc = "A pair of thick, long, yellow rubber gloves, designed to protect the wearer from the splash of industrial strength cleaners. Not electrically insulated."
```
Comment (FlamingLily) [inline code/modules/clothing/gloves/miscellaneous.dm:149]:
```suggestion
	desc = "A pair of thick, long, yellow rubber gloves, designed to protect the wearer from the splash of industrial strength cleaners. Not certified for electrical work."
```

**Review by FlamingLily — COMMENTED** [review]
lazy commits

File: code/modules/clothing/gloves/miscellaneous.dm (hunk 4 above)
Comment (FlamingLily) [reply code/modules/clothing/gloves/miscellaneous.dm]: Yeah fair

**Comment by FlamingLily** [comment]
I'm not sure why that one check is failing

File: html/changelogs/flaminglily-gloves.yml [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+# Your name.
+author: FlamingLily
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
```
Comment (FlamingLily) [inline html/changelogs/flaminglily-gloves.yml:38]:
```suggestion
# Aaaaaaaaaaaaaaaaaaalso, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
```

**Comment by FlamingLily** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Review by alsoandanswer — DISMISSED** [review]
I think it would be better to just make a box that spawns in the closet with all the different xeno cleaning gloves rather than putting them in the loadout. Should make the loadout less clogged up.

**Comment by FlamingLily** [comment]
fair

**Pull request merged**